demo/main.py

```diff
@@ -136,14 +137,15 @@ class Section:
   Section(
     name="Form",
     examples=[
+      Example(name="autocomplete"),
       Example(name="button"),
       Example(name="checkbox"),
       Example(name="input"),
-      Example(name="textarea"),
       Example(name="radio"),
       Example(name="select_demo"),
       Example(name="slide_toggle"),
       Example(name="slider"),
+      Example(name="textarea"),
```

Comment:
I know you're trying to alphabetize, but I intentionally put textarea next to input (same as the docs) since they are similar functionality-wise. I think it's OK to not be strictly alphabetical.

Reply:
That's fair. Updated.